src/components/Select/Select.stories.mdx

```diff
@@ -0,0 +1,98 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { useState, useEffect } from 'react';
+import Select from './Select';
+import { userEvent, waitFor, screen} from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+
+<Meta
+  title="Base Components/Select"
+  argTypes={{
+    options: {
+      control: 'array',
+      description: 'test of options to choose from',
+      defaultValue: ['Danish', 'English', 'French', 'German', 'Norwegian'],
+    },
+    option: {
+      control: 'select',
+      description: 'Currently selected option',
+      defaultValue: 'English',
+      options: ['Danish', 'English', 'French', 'German', 'Norwegian'],
+    },
+    setOption: {
+      action: true,
+    },
+    fixedOptions: {
+      control: 'array',
+      description:
+        'Prioritised options which are sorted to the top of the list',
+      defaultValue: ['English', 'Norwegian'],
+    },
+    loading: {
+      control: 'radio',
+      description: 'Indicates that more options are being loaded.',
+      defaultValue: 'false',
+      options: [true, false],
+    },
+  }}
+  component={Select}
+/>
+
+export const Template = (args) => <Select {...args} />;
+
+# Select
+
+<Canvas>
+  <Story
+    name="Languages"
+    height='350px'
+    play={async ({ args }) => {
+      await userEvent.click(screen.getByRole('button'));
```

Comment:
Interaction tests to ensure functionality, check the "Interactions" tab in Storybook.